chart/iam-runtime-infratographer/templates/_configmap.tpl

```diff
@@ -1,6 +1,6 @@
 {{- define "iam-runtime-infratographer.configmap" }}
 {{- $values := (index .Subcharts "iam-runtime-infratographer").Values -}}
-{{- $defaultConfig := dict "server" (dict "socketPath" "/var/iam-runtime/runtime.sock") }}
+{{- $defaultConfig := dict "server" (dict "socketPath" (printf "%sruntime.sock" .Values.socketVolumeMountPath)) }}
```

Comment:
This should have a `/` between the path and the socket file. If socketVolumeMountPath is defined without a trailing `/` this would have issues.